dynamodump.py

```diff
@@ -39,48 +37,68 @@
 MAX_RETRY = 6
 LOCAL_REGION = "local"
 LOG_LEVEL = "INFO"
-DATA_DUMP = "dump"
+DATA_DUMP = "dynamodump"
+OUTPUT_DIR = "/tmp/dynamorestore/"
 RESTORE_WRITE_CAPACITY = 25
 THREAD_START_DELAY = 1  # seconds
 CURRENT_WORKING_DIR = os.getcwd()
-DEFAULT_PREFIX_SEPARATOR = "-"
+DEFAULT_PREFIX_SEPARATOR = ","
```

Comment:
This is going to be a breaking change in default behaviour, I'll have to go through more in detail again